### ISSUE TITLE
find better way of dealing with images than urlencoding in the json

### ISSUE THREAD

**Issue by jspaaks** [research-software-directory/research-software-directory#619]
Currently the images are either urlencoded into the json or they are external via hyperlinks. We should let the RSD have its own copies of any images that it uses.

**Comment by sverhoeven** [comment]
Agree, having 2 different ways of handling images is confusing and sup-optimal.

We are now providing some of the images as a base64 string in /api calls.

Also the project images used on the pages excluding the main figure on the project page are tiny (84x47 pixels), while we are using the full resolution image. Having a thumbnail besides the original will help reduce page load times.

**Comment by sverhoeven** [comment]
As of #676 and #696 we are moving replacing the imageUrls with blobs in the db. So we will have a single way to handle images, but having the blobs inlined in JSON is not optimal and still needs to be addressed.

**Comment by sverhoeven** [comment]
The project index page now is 19Mb in size (mainly due to the base64 image stirings) and takes 1.5 seconds to render in my browser. 
This is not very friendly.
